### PR TITLE
test: cypress - updated click actions in tests

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
Updated steps related to click action in **Listv2_onItemClick_spec.js**

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/Listv2_onItemClick_spec.js [hunk 1]
```diff
@@ -112,20 +110,14 @@ describe("List widget v2 onItemClick", () => {
...
     agHelper.WaitUntilToastDisappear("ListWidget_Blue_0");
 
-    cy.get(widgetsPage.toggleOnClick).click({ force: true });
-    cy.get(".t--property-control-onclick").then(($el) => {
-      cy.updateCodeInput($el, "{{clearStore()}}");
-    });
-    cy.wait(1000);
+    propPane.ToggleJSMode("onClick");
+    propPane.UpdatePropertyFieldValue("onClick", "{{clearStore()}}");
+    agHelper.Sleep(1000);
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/Listv2_onItemClick_spec.js:118]: Why is `EnterJSContext` not used? any reason?

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/Listv2_onItemClick_spec.js [hunk 2]
```diff
@@ -112,20 +110,14 @@ describe("List widget v2 onItemClick", () => {
...
 
-    cy.get(`${widgetSelector("Button1")} button`)
-      .first()
-      .click({ force: true });
-    validateToastDoestExist();
+    agHelper.GetNClick(`${locators._widgetByName("Button1")} button`, 0, true);
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/Listv2_onItemClick_spec.js:120]: Why is `ClickButton` not used here?

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/Listv2_onItemClick_spec.js [hunk 3]
```diff
@@ -112,20 +110,14 @@ describe("List widget v2 onItemClick", () => {
       draggableWidgets.CONTAINER,
     );
 
-    cy.get(`${widgetSelector("Button1")} button`)
-      .first()
-      .click({ force: true });
+    agHelper.GetNClick(`${locators._widgetByName("Button1")} button`, 0, true);
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/Listv2_onItemClick_spec.js:113]: Why is `ClickButton` not used here?

**Review by Aishwarya-U-R — COMMENTED** [review]
Add the spec name to the PR description

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/Listv2_onItemClick_spec.js (hunk 1 above)
Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/Listv2_onItemClick_spec.js:118]: done

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/Listv2_onItemClick_spec.js (hunk 2 above)
Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/ListV2/Listv2_onItemClick_spec.js]: done

**Comment by Aishwarya-U-R** [comment]
@NandanAnantharamu - [Listv2_onItemClick_spec](https://cloud.cypress.io/projects/eyxvp8/runs/37926/test-results?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%7B%22label%22%3A%22Regression%2FClientSide%2FWidgets%2FListV2%2FListv2_onItemClick_spec.js%22%2C%22value%22%3A%22%5B%5C%22dff71055-b03c-4f2f-9b12-be9e6d029534%5C%22%5D%22%7D%5D&statuses=%5B%7B%22value%22%3A%22FAILED%22%2C%22label%22%3A%22FAILED%22%7D%5D&testingTypesEnum=%5B%5D) has failed in CI run. Please check.

**Comment by NandanAnantharamu** [comment]
> @NandanAnantharamu - [Listv2_onItemClick_spec](https://cloud.cypress.io/projects/eyxvp8/runs/37926/test-results?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%7B%22label%22%3A%22Regression%2FClientSide%2FWidgets%2FListV2%2FListv2_onItemClick_spec.js%22%2C%22value%22%3A%22%5B%5C%22dff71055-b03c-4f2f-9b12-be9e6d029534%5C%22%5D%22%7D%5D&statuses=%5B%7B%22value%22%3A%22FAILED%22%2C%22label%22%3A%22FAILED%22%7D%5D&testingTypesEnum=%5B%5D) has failed in CI run. Please check.

updated test lets wait for CI run

**Comment by Aishwarya-U-R** [comment]
> > @NandanAnantharamu - [Listv2_onItemClick_spec](https://cloud.cypress.io/projects/eyxvp8/runs/37926/test-results?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%7B%22label%22%3A%22Regression%2FClientSide%2FWidgets%2FListV2%2FListv2_onItemClick_spec.js%22%2C%22value%22%3A%22%5B%5C%22dff71055-b03c-4f2f-9b12-be9e6d029534%5C%22%5D%22%7D%5D&statuses=%5B%7B%22value%22%3A%22FAILED%22%2C%22label%22%3A%22FAILED%22%7D%5D&testingTypesEnum=%5B%5D) has failed in CI run. Please check.
> 
> updated test lets wait for CI run

Did the tests pass locally after this update? Can you please post screenshot of passing tests?

**Comment by NandanAnantharamu** [comment]
> > > @NandanAnantharamu - [Listv2_onItemClick_spec](https://cloud.cypress.io/projects/eyxvp8/runs/37926/test-results?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%7B%22label%22%3A%22Regression%2FClientSide%2FWidgets%2FListV2%2FListv2_onItemClick_spec.js%22%2C%22value%22%3A%22%5B%5C%22dff71055-b03c-4f2f-9b12-be9e6d029534%5C%22%5D%22%7D%5D&statuses=%5B%7B%22value%22%3A%22FAILED%22%2C%22label%22%3A%22FAILED%22%7D%5D&testingTypesEnum=%5B%5D) has failed in CI run. Please check.
> > 
> > 
> > updated test lets wait for CI run
> 
> Did the tests pass locally after this update? Can you please post screenshot of passing tests?

**Comment by Aishwarya-U-R** [comment]
I see only like this:
Screenshot 2023-07-27 at 8 39 41 PM

**Comment by NandanAnantharamu** [comment]
> I see only like this: Screenshot 2023-07-27 at 8 39 41 PM

Screenshot 2023-07-27 at 8 38 01 PM

**Pull request merged**